mws/models/base.py

```diff
@@ -1,12 +1,41 @@
 """Base models for datatypes used in MWS."""
 
 from abc import ABC, abstractmethod
+from os import stat
+from typing import Any, Iterable, Mapping, Union
+from enum import Enum
+
+from mws.utils import flat_param_dict
 
 
 class MWSDataType(ABC):
     """Base class for data type models used for MWS requests."""
 
+    def __repr__(self):
+        output = f"<{self.__class__.__name__}("
+        values = [f"{key}={repr(val)}" for key, val in self.__dict__.items()]
+        output += ", ".join(values)
+        output += ")>"
+        return output
+
     @abstractmethod
-    def to_dict(self) -> dict:
-        """Returns a flattened dict of parameters suitable for an MWS request."""
+    def params_dict(self) -> dict:
+        """Returns a dict of this model's parameters suitable for an MWS request."""
         pass
+
+    def to_params(self, prefix: str = "") -> dict:
+        params_dict = self.params_dict()
+        return flat_param_dict(params_dict, prefix=prefix)
+
+    @staticmethod
+    def flat_param_dict(value: Union[str, Mapping, Iterable], prefix: str = "") -> dict:
+        return flat_param_dict(value, prefix=prefix)
```

Comment:
Simple static method to provide access to `flat_param_dict` within any subclass of `MWSDataType`.

Actually, probably not being used anywhere at the moment, but having it there as an easily-accessible tool is a nice-to-have.